src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -153,6 +154,9 @@ public static class Configuration
 	// This flag indicates whether to ignore unexpected elements and attributes in the metadata document and generate
 	// the client code if any. The value must be set to true or false.
 	public const bool IgnoreUnexpectedElementsAndAttributes = true;
+
+	// (Optional) Custom http headers in this format: {headerName:value},{headerName:value}

```

Comment:
where's from this pattern " {key:value},{key:value}  "?

Reply:
@mikepizzo gave a detailed explanation in a previous incomplete PR https://github.com/OData/odata.net/pull/1348#issuecomment-447116560

Reply:
[https://github.com/OData/odata.net/pull/1348#issuecomment-447116560](https://github.com/OData/odata.net/pull/1348#issuecomment-447116560)

---
In reply to: [368252373](https://github.com/OData/odata.net/pull/1637#discussion_r368252373) [](ancestors = 368252373)

Reply:
Thanks, but how about if the value has ":" itself? should we escaped it?

---
In reply to: [368404438](https://github.com/OData/odata.net/pull/1637#discussion_r368404438) [](ancestors = 368404438)